server/src/test/java/io/deephaven/server/runner/DeephavenApiServerTestBase.java

```diff
@@ -62,6 +65,9 @@ interface Builder {
             @BindsInstance
             Builder withServerConfig(ServerConfig serverConfig);
 
+            @BindsInstance
+            Builder withConfiguration(Configuration config);;
+
```

Comment:
I think this is safe to delete (and import above call below), or at least remove the double `;`